### PR TITLE
JA: translation /ja/install/gpu.md

### PR DESCRIPTION
**Pull request by yukinagae**
This PR is just a split small version of this #442 for reviewers to check the content of PR easily.

@sfujiwara You might have already reviewed site/ja/install/gpu.md. I've made some changes on the gpu.md file to follow up the latest master branch. You can check the below diff:

```
$ git diff 8cdbb909:site/en/install/gpu.md 7022de7:site/en/install/gpu.md
diff --git a/site/en/install/gpu.md b/site/en/install/gpu.md
index 1c10acb3c..1633d5034 100644
--- a/site/en/install/gpu.md
+++ b/site/en/install/gpu.md
@@ -21,13 +21,13 @@ GPU support, choose a stable or development package:
 <code class="devsite-terminal">pip install tf-nightly-gpu  # preview</code>
 </pre>

-### TensorFlow 2.0 Alpha
+### TensorFlow 2.0 Beta

-[TensorFlow 2.0 Alpha](../alpha) is available for testing with GPU support. To
-install:
+[TensorFlow 2.0 Beta](https://www.tensorflow.org/beta) is available for testing
+with GPU support. To install:

 <pre class="prettyprint lang-bsh">
-<code class="devsite-terminal">pip install tensorflow-gpu==2.0.0-alpha0</code>
+<code class="devsite-terminal">pip install tensorflow-gpu==2.0.0-beta0</code>
 </pre>

 ## Hardware requirements
@@ -43,7 +43,8 @@ The following GPU-enabled devices are supported:
 The following NVIDIA® software must be installed on your system:

 * [NVIDIA® GPU drivers](https://www.nvidia.com/drivers){:.external} —CUDA 10.0 requires 410.x or higher.
-* [CUDA® Toolkit](https://developer.nvidia.com/cuda-zone){:.external} —TensorFlow supports CUDA 10.0 (TensorFlow >= 1.13.0)
+* [CUDA® Toolkit](https://developer.nvidia.com/cuda-toolkit-archive){:.external}
+  —TensorFlow supports CUDA 10.0 (TensorFlow >= 1.13.0)
 * [CUPTI](http://docs.nvidia.com/cuda/cupti/){:.external} ships with the CUDA Toolkit.
 * [cuDNN SDK](https://developer.nvidia.com/cudnn){:.external} (>= 7.4.1)
 * *(Optional)* [TensorRT 5.0](https://docs.nvidia.com/deeplearning/sdk/tensorrt-install-guide/index.html){:.external}
@@ -124,7 +125,7 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
 <code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
 <code class="devsite-terminal">sudo apt-get update</code>

-# Install NVIDIA Driver
+# Install NVIDIA driver
 # Issue with driver install requires creating /usr/lib/nvidia
 <code class="devsite-terminal">sudo mkdir /usr/lib/nvidia</code>
 <code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-410</code>
@@ -157,6 +158,12 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
 <code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
 <code class="devsite-terminal">sudo apt update</code>

+# Install the NVIDIA driver
+# Issue with driver install requires creating /usr/lib/nvidia
+<code class="devsite-terminal">sudo mkdir /usr/lib/nvidia</code>
+<code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-410</code>
+# Reboot. Check that GPUs are visible using the command: nvidia-smi
+
 # Install CUDA and tools. Include optional NCCL 2.x
 <code class="devsite-terminal">sudo apt install cuda9.0 cuda-cublas-9-0 cuda-cufft-9-0 cuda-curand-9-0 \
     cuda-cusolver-9-0 cuda-cusparse-9-0 libcudnn7=7.2.1.38-1+cuda9.0 \
@@ -186,5 +193,6 @@ environmental variable. For example, if the CUDA Toolkit is installed to
 <pre class="devsite-click-to-copy">
 <code class="devsite-terminal tfo-terminal-windows">SET PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\bin;%PATH%</code>
 <code class="devsite-terminal tfo-terminal-windows">SET PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\extras\CUPTI\libx64;%PATH%</code>
+<code class="devsite-terminal tfo-terminal-windows">SET PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0\include;%PATH%</code>
 <code class="devsite-terminal tfo-terminal-windows">SET PATH=C:\tools\cuda\bin;%PATH%</code>
 </pre>
```

### CODE REVIEW

File: site/ja/install/gpu.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+* NVIDIA® GPU card with CUDA® Compute Capability 3.以降
+  [CUDA対応のGPUカード](https://developer.nvidia.com/cuda-gpus){:.external}を参照してください。
+
+
+## ソフトウェア要件
+
+以下のNVIDIA®ソフトウェアがお使いのシステムにインストールされている必要があります:
+
+* [NVIDIA® GPドライバー](https://www.nvidia.com/drivers){:.external} —CUDA 10は 410.x 以降を必要とします
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: This line should be:

* [NVIDIA® GPUドライバー](https://www.nvidia.com/drivers){:.external} —CUDA 10.0 は 410.x 以降を必要とします

File: site/ja/install/gpu.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+<code class="devsite-terminal">sudo apt-get update</code>
+<code class="devsite-terminal">wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt-get update</code>
+
+# NVIDIAドライバーをインストールする
+# ドライバーのインストールに関する問題は /usr/lib/nvidia の作成を必要とします
+<code class="devsite-terminal">sudo mkdir /usr/lib/nvidia</code>
+<code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-410</code>
+# 再起動。コマンドを使用してGPUが表示されていることを確認します: nvidia-smi
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: How about:

# 再起動します。nvidia-smi コマンドを使用してGPUが表示されていることを確認します

File: site/ja/install/gpu.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+<code class="devsite-terminal">sudo apt install ./cuda-repo-ubuntu1604_9.1.85-1_amd64.deb</code>
+<code class="devsite-terminal">wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt update</code>
+
+# NVIDIAドライバーをインストールする
+# ドライバーのインストールに関する問題は /usr/lib/nvidia の作成を必要とします
+<code class="devsite-terminal">sudo mkdir /usr/lib/nvidia</code>
+<code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-410</code>
+# 再起動。コマンドを使用してGPUが表示されていることを確認します: nvidia-smi
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: same as above.

"# 再起動します。nvidia-smi コマンドを使用してGPUが表示されていることを確認します"

File: site/ja/install/gpu.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+<code class="devsite-terminal">pip install tf-nightly-gpu  # preview</code>
+</pre>
+
+### TensorFlow 2.0 Beta
+
+GPUサポートのテスト用に[TensorFlow 2.0 Beta](https://www.tensorflow.org/beta)が利用可能です。
+インストールするには:
+
+<pre class="prettyprint lang-bsh">
+<code class="devsite-terminal">pip install tensorflow-gpu==2.0.0-beta0</code>
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: Original source changed:

beta0 -> beta1

File: site/ja/install/gpu.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+</pre>
+
+## ハードウェア要件
+
+以下のGPU対応デバイスがサポートされています:
+
+* NVIDIA® GPU card with CUDA® Compute Capability 3.以降
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: This line should be fixed as below:

3.以降 -> 3.5 以降

File: site/ja/install/gpu.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+<code class="devsite-terminal">sudo dpkg -i cuda-repo-ubuntu1804_10.0.130-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/7fa2af80.pub</code>
+<code class="devsite-terminal">sudo apt-get update</code>
+<code class="devsite-terminal">wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64/nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb</code>
+<code class="devsite-terminal">sudo apt-get update</code>
+
+# NVIDIAドライバーをインストールする
+<code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-driver-410</code>
+# 再起動。コマンドを使用してGPUが表示されていることを確認します: nvidia-smi
```
Comment (masa-ita) [inline site/ja/install/gpu.md]: Same line.

"# 再起動します。nvidia-smi コマンドを使用してGPUが表示されていることを確認します"

File: site/ja/install/gpu.md (hunk 1 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: I'll fix typo, thanks :)

File: site/ja/install/gpu.md (hunk 4 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: I didn't notice it. Thanks!

File: site/ja/install/gpu.md (hunk 5 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: It looks like a typo, thanks :)

File: site/ja/install/gpu.md (hunk 2 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: Thanks for the review. How about the below translation?

```
# 再起動します。次のコマンドを使用してGPUが表示されていることを確認します: nvidia-smi 
```

I prefer not to change a lot from the original sentence when translating it. But I reckon your suggestion is much better as a Japanese sentence than mine ...

File: site/ja/install/gpu.md (hunk 3 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: same as above: https://github.com/tensorflow/docs/pull/679#discussion_r296105441

File: site/ja/install/gpu.md (hunk 6 above)
Comment (yukinagae) [reply site/ja/install/gpu.md]: same as above: https://github.com/tensorflow/docs/pull/679#discussion_r296105441

**Pull request merged**